### PR TITLE
Document appendable mmap

### PR DESCRIPTION
**Pull request by timvisee**
Extends aba5b3ed0bef8febe09715ec72d9d0e51be61758.

Document support for appenadble mmap's. I'm not sure to what detail we want to describe this, but here is my version describing it in a few sentences.

This describes:

- at collection creation, that users can use `on_disk = true` to enable appendable mmap's
- in the how to section, to use appendable mmap's when ingesting a large amount of data

### CODE REVIEW

File: qdrant/v1.2.x/collections.md [hunk 1]
```diff
@@ -59,7 +59,14 @@ In addition to the required options, you can also specify custom values for the
 
 Default parameters for the optional collection parameters are defined in [configuration file](https://github.com/qdrant/qdrant/blob/master/config/config.yaml).
 
-See [schema definitions](https://qdrant.github.io/qdrant/redoc/index.html#operation/create_collection) and a [configuration file](https://github.com/qdrant/qdrant/blob/master/config/config.yaml) for more information about collection parameters.
+See [schema definitions](https://qdrant.github.io/qdrant/redoc/index.html#operation/create_collection) and a [configuration file](https://github.com/qdrant/qdrant/blob/master/config/config.yaml) for more information about collection and vector parameters.
+
+*Available as of v1.2.0*
+
+The `on_disk` parameter can be set in the vector configuration. If true, all
+vectors will be stored on disk at all times. This will enable the use of
```
Comment (agourlay) [inline qdrant/v1.2.x/collections.md:67]: I find our usage of `store` in the docs more and more confusing.
Are there times where the vectors are not stored on disk?

My understanding is that the vectors are always persisted on disk but they are sometimes loaded in memory for performance reason.

WDYT?

Comment (timvisee) [reply qdrant/v1.2.x/collections.md:67]: > Are there times where the vectors are not stored on disk?

That is a very good point. You're right.

The thing is, that with mmap (`on_disk = true`), vectors are also in memory (most of the time).

I'm not sure how to phrase this better. I don't think the following makes it clear:

> Normally vectors are primarily in RAM. With appendable mmap vectors are primarily on disk.

File: qdrant/v1.2.x/how_to.md [hunk 2]
```diff
@@ -533,6 +533,20 @@ client.update_collection(
 )
 ```
 
+### Appendable mmap
```
Comment (generall) [inline qdrant/v1.2.x/how_to.md:536]: how-to/appendable-mmap doesn't really match the semantics of this section

Comment (timvisee) [reply qdrant/v1.2.x/how_to.md]: I've updated it. Is this better?

https://github.com/qdrant/docs/blob/428ab37d56ce6a41e033331fa965c27a03e308f9/qdrant/v1.2.x/how_to.md?plain=1#L536-L554

File: qdrant/v1.2.x/collections.md (hunk 1 above)
Comment (timvisee) [reply qdrant/v1.2.x/collections.md]: Agreed in private chat that this is better. Marking as resolved. :+1:

**Review by agourlay — APPROVED** [review]

**Comment by davidmyriel** [comment]
already merged to the new repo

**Pull request closed without merging**